tests.py

```diff
@@ -0,0 +1,13 @@
+from tvl2018 import *
+# to specify rate, add rate = your_rate_value in the parameters
+# to specify output_path, add output_path = your_output_path in the parameters
+
+# Add more tests
+
+def test_loudness_model():
```

Comment:
This is not the right way to do the test... but let's keep this for now, and once we check things in I can show you the right pattern.